src/components/App.tsx

```diff
@@ -2586,8 +2588,8 @@ class App extends React.Component<ExcalidrawProps, AppState> {
       );
 
       /* If arrow is pre-arrowheads, it will have undefined for both start and end arrowheads.
-			 If so, we want it to be null for start and "arrow" for end. If the linear item is not 
-			 an arrow, we want it to be null for both. Otherwise, we want it to use the 
+			 If so, we want it to be null for start and "arrow" for end. If the linear item is not
+			 an arrow, we want it to be null for both. Otherwise, we want it to use the
 			 values from appState. */
```

Comment:
How come we have tabs here :D

Reply:
That's strange... I'm not sure, but I did have some issues with Prettier while making this fix.

Reply:
Hmm... I'm a bit confused. Should I revert this change?

Reply:
Convert it to spaces

Reply:
Prettier is not changing much in the comments as far as I know.

Reply:
there's an eslint rule preventing mixed tabs/spaces IIRC, if anyone wants to spend time on it :)

Reply:
Will take a look.. but let's remove them manually here for now.

Reply:
> there's an eslint rule preventing mixed tabs/spaces IIRC, if anyone wants to spend time on it :)

https://eslint.org/docs/rules/no-mixed-spaces-and-tabs doesn't seem to affect comments as well.

Reply:
It's converted it so spaces :)